startup_scripts/010_groups.py

```diff
@@ -19,17 +20,4 @@
       user.groups.add(group)
 
   yaml_permissions = group_details.get('permissions', [])
-  if yaml_permissions:
-    subject = group.permissions
-    subject.clear()
-    for yaml_permission in yaml_permissions:
-      if '*' in yaml_permission:
-        permission_filter = '^' + yaml_permission.replace('*','.*') + '$'
-        permissions = Permission.objects.filter(codename__iregex=permission_filter)
-        print("  ⚿ Granting", permissions.count(), "permissions matching '" + yaml_permission + "'")
-      else:
-        permissions = Permission.objects.filter(codename=yaml_permission)
-        print("  ⚿ Granting permission", yaml_permission)
-
-      for permission in permissions:
-        subject.add(permission)
+  set_perms_from_yaml(subject=group.permissions, permissions=yaml_permissions)
```

Comment:
Following the other suggestions, this would become that:

```suggestion
  set_permissions(group.permissions, yaml_permissions)
```

Reply:
Done